components/collections/codepointtrie_builder/src/wasm.rs

```diff
@@ -6,17 +6,12 @@ use crate::CodePointTrieBuilder;
 use crate::CodePointTrieBuilderData;
 use icu_collections::codepointtrie::TrieType;
 use icu_collections::codepointtrie::TrieValue;
-use lazy_static::lazy_static;
+use std::io::{Read, Write};
 use wasmer::{Instance, Module, Store};
-use wasmer_wasi::{Pipe, WasiState};
+use wasmer_wasi::{Pipe, WasiError, WasiState};
 
 const WASM_BYTES: &[u8] = include_bytes!("../list_to_ucptrie.wasm");
 
-lazy_static! {
```

Comment:
Nit: If you are not using lazy_static any more, please remove it from cargo.toml